services/frontend/src/store/index.js

```diff
@@ -19,8 +20,10 @@ export default new Vuex.Store({
6472168, 0.012666195631027222, 0.08199697732925415, 0.08714157342910767, 0.020994842052459717, 0.011242300271987915, 0.0038233399391174316, 0.00453612208366394, 0.014768421649932861, 0.005400240421295166, 0.003352820873260498, 0.0032306909561157227, 0.003724008798599243, 0.00835302472114563, 0.014870375394821167, 0.050987958908081055, 0.34861382842063904, 0.1826949119567871, 0.05069825053215027, 0.0222322940826416, 0.07498842477798462, 0.06072497367858887, 0.02902117371559143, 0.0250871479511261, 0.01841825246810913, 0.004292845726013184, 0.00492250919342041, 0.013436973094940186, 0.028086692094802856, 0.02277711033821106, 0.022185295820236206, 0.00572812557220459, 0.00393223762512207, 0.012085795402526855, 0.019610822200775146, 0.013373464345932007, 0.013629913330078125, 0.008064419031143188, 0.002899169921875, 0.0034671127796173096, 0.011363506317138672, 0.013469904661178589, 0.014147758483886719, 0.012260109186172485, 0.009284347295761108, 0.005585730075836182, 0.0030487775802612305, 0.0035476386547088623, 0.004796117544174194, 0.006975084543228149, 0.008821964263916016, 0.011093080043792725, 0.0018616914749145508, 0.001266777515411377, 0.0033701658248901367, 0.0051246583461761475, 0.004507631063461304, 0.004613429307937622, 0.008398771286010742, 0.003271430730819702, 0.0007448494434356689, 0.0014591515064239502, 0.0014681220054626465, 0.0022083520889282227, 0.0037546753883361816, 0.01028674840927124, 0.005567222833633423, 0.005117267370223999, 0.002554178237915039, 0.008505135774612427, 0.011288493871688843, 0.0026982128620147705, 0.0018385350704193115, 0.0032823681831359863, 0.0026774704456329346, 0.0030251145362854004, 0.003700166940689087, 0.0030539333820343018, 0.04479125142097473, 0.05470985174179077, 0.04300948977470398, 0.01989874243736267, 0.00938868522644043, 0.0025010406970977783, 0.004980981349945068, 0.06438800692558289, 0.44057172536849976, 0.27556610107421875, 0.011737614870071411, 0.002435743808746338, 0.0014040470123291016, 0.006583809852600098, 0.0760832130908966, 0.00637546181678772, 0.003078550100326538, 0.0021862387657165527, 0.004487425088882446, 0.058743685483932495, 0.0885833203792572, 0.02445504069328308, 0.009481340646743774, 0.0038950443267822266, 0.004348307847976685, 0.0029076039791107178, 0.0015988647937774658, 0.008667916059494019, 0.0028187036514282227, 0.001346677541732788, 0.0009747445583343506, 0.013713091611862183, 0.012329459190368652, 0.003603994846343994, 0.006424993276596069, 0.0014544427394866943, 0.008217662572860718, 0.012168645858764648, 0.002413123846054077, 0.003259599208831787, 0.0016272366046905518, 0.0012287795543670654, 0.00694584846496582, 0.024220138788223267, 0.09544476866722107, 0.02614825963973999, 0.008856624364852905, 0.00785788893699646, 0.0017541050910949707, 0.0029064714908599854, 0.0017959773540496826, 0.0017816722393035889, 0.004655629396438599, 0.23912471532821655, 0.3461966812610626, 0.7516708374023438, 0.7929297685623169, 0.9548741579055786, 0.910999059677124, 0.6750947833061218]
```

Comment:
This single line of code breaks my chrome tab. Also we are not mocking analysis data anymore so get rid of it.